config/samples/gcp/gcp_openshift_route.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: sample-gcp
+spec:
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: misalunkgcp.qe.gcp.devcluster.openshift.com
+  provider:
+    type: GCP
+  source:
+    hostnameAnnotation: Allow
+    openshiftRouteOptions:
```

Comment:
Can you please add a comment explaining why we need this field? This would be useful for the users.